README.md

```diff
@@ -30,7 +30,9 @@ The
 [`api/prometheus` directory](https://github.com/prometheus/client_golang/tree/master/api/prometheus)
 contains the client for the
 [Prometheus HTTP API](http://prometheus.io/docs/querying/api/). It allows you
-to write Go applications that query time series data from a Prometheus server.
+to write Go applications that query time series data from a Prometheus
+server. It is in an early stage and further development is currently on hold
```

Comment:
But how can we express that there will be nobody reviewing PRs?

And how does the "it's alpha" not reflect on the whole client with the same argument you used above? (I don't think this misunderstanding is likely. I'm just trying to understand your line of reasoning.)